zio-json/shared/src/test/scala/zio/json/ConfigurableDeriveCodecSpec.scala

```diff
@@ -88,6 +109,27 @@ object ConfigurableDeriveCodecSpec extends ZIOSpecDefault {
           assertTrue(
             jsonAST.as[ClassWithFields].toOption.get == expectedObj
           )
+        },
+        test("do not write nulls by default") {
+          val jsonAST     = Json.Obj()
+          val expectedObj = OptionalField(None)
+
+          implicit val codec: JsonCodec[OptionalField] = DeriveJsonCodec.gen
+
+          assertTrue(
+            jsonAST.as[OptionalField].toOption.get == expectedObj,
+            expectedObj.toJsonAST == Right(jsonAST)
+          )
+        },
+        test("write empty collections by default") {
+          case class EmptySeq(a: Seq[Int])
+
+          val expectedStr = """{"a":[]}"""
+          val expectedObj = EmptySeq(Seq.empty)
+
+          implicit val codec: JsonCodec[EmptySeq] = DeriveJsonCodec.gen
+
+          assertTrue(expectedStr.fromJson[EmptySeq].toOption.get == expectedObj, expectedObj.toJson == expectedStr)
```

Comment:
Shouldn't be assertion for `expectedObj.toJsonAST` here instead of  `expectedObj.toJson`?

Reply:
yes, fixed!